lib/util/http/github.ts

```diff
@@ -345,6 +345,7 @@ export class GithubHttp extends HttpBase<GithubHttpOptions> {
     const result = await super.requestJsonUnsafe<T>(method, opts);
     if (httpOptions.paginate) {
       delete httpOptions.cacheProvider;
+      httpOptions.memCache = false;
```

Comment:
don't we need it on other http platform implementations too?

Reply:
Maybe, I'll double check. Was searching for `cacheProvider` usage only...